mappings/net/minecraft/block/DoorBlock.mapping

```diff
@@ -70,7 +70,7 @@ CLASS biw net/minecraft/block/DoorBlock
 	METHOD a appendProperties (Lbqj$a;)V
 		ARG 1 builder
 	METHOD c getRenderLayer ()Lbbu;
-	METHOD d getClosingSound ()I;
-	METHOD e getOpeningSound ()I;
+	METHOD d getClosingSound ()Lxr;
+	METHOD e getOpeningSound ()Lxr;
```

Comment:
Are these methods actaully returning an int or `xr`?

Can these mappings be checked to see if they are still valid.

Reply:
They get passed into `World#fireWorldEvent` so should be `()I`

Reply:
Oh my bad I didn't get what asie requested first

Reply:
`getClosingWorldEvent`?